amundsen_application/static/js/ducks/api/metadata/v0.ts

```diff
@@ -140,9 +142,22 @@ export function metadataUpdateColumnDescription(description, columnIndex, tableD
   }
 }
 
-
 export function metadataGetLastIndexed() {
   return axios.get(`${API_PATH}/get_last_indexed`).then((response) => {
     return response.data.timestamp;
   });
 }
+
+export function metadataGetPreviewData(action: GetPreviewDataRequest) {
```

Comment:
I've noticed the parameters are sometimes reducer actions and other times they are picked out from picked out of the reducer action.
E.g.
```
export function metadataUpdateColumnDescription(description, columnIndex, tableData) { ... }
yield call(metadataUpdateColumnDescription, action.newValue, action.columnIndex, state.tableMetadata.tableData);
```
vs 
```
export function metadataUpdateColumnDescription(action: updateColumnDescriptionRequest) { ... }
yield call(metadataUpdateColumnDescription, action);
```

Functionally these are equivalent but we should pick a pattern and try to be consistent. I have a slight preference on the latter but could go either way. Thoughts?

Reply:
Agreed. I actually have a preference right now for the first option because the second option assumes a one to one mapping between actions and these helper methods, which is currently the case but I don't know if it will always be. But perhaps it's also a better pattern to keep the one to one mapping. Can you file an internal ticket in our ever-growing improvements epic? 

Open to discussing offline this week so we can decrease tech debt by sticking to one pattern for all upcoming work.